error: explicit error: err
 --> inner_error_2.jsonnet:1:30
  |
1 | std.__compare([0, [1]], [0, [error "err"]])
  |                              ^^^^^^^^^^^
note: while comparing array item 0
note: while comparing array item 1
note: while evaluating call to `__compare`
 --> inner_error_2.jsonnet:1:1
  |
1 | std.__compare([0, [1]], [0, [error "err"]])
  | -------------------------------------------
note: during top-level value evaluation

